src/main/java/wolox/training/models/Users.java

```diff
@@ -0,0 +1,74 @@
+package wolox.training.models;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Users {
```

Comment:
As a rule of thumb, model classes are singular (User), is there a reason why you created it in plural? If it's because of the collision with postgres, you can add this annotation: `@Table(name = "users")` 😄